package.json

```diff
@@ -30,7 +30,7 @@
   },
   "dependencies": {
     "@dcos/http-service": "0.2.1",
-    "@dcos/mesos-client": "0.1.10",
+    "@dcos/mesos-client": "0.2.0",
```

Comment:
updated the PR to include the changes https://github.com/dcos-labs/mesos-client/pull/7